src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -34,6 +35,14 @@ class ExternalIssueForm extends AsyncComponent {
     onSubmitSuccess: PropTypes.func.isRequired,
   };
 
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      showAllFields: false,
+    };
```

Comment:
`static state` again

Reply:
gonna remove these so I know what to do and what not to